content/PreparedStatement/PreparedStatement.q.md

```diff
@@ -0,0 +1,1707 @@
+# Prepared Statement
+
++++
+
+### Part 1: Sample Code Analysis
+
+:::
+
+/// type=REPL, readonly=true, filename=[connection.php,student.php], init=[commands/PreparedStatement/InsertValuesBerryVikkiIntoStudentTable.sql]
+
+```php
+// connection.php
+<?php
+    $host = 'localhost';
+    $db = 'LibraryDB';
+    $port = '5432';
+    $username = 'postgres';
+    $password = 'Admin01';
+
+    try {
+        $conn = new PDO("pgsql:host=$host;port=$port;dbname=$db;user=$username;password=$password");
+        if ($conn) {
+            echo "Successfully connected to the database. ";
+        }
+    } catch (Exception $e) {
+        echo "Unable to establish a connection."; 
+    }
+?>
+```
+
+```php
+// student.php
+<?php
+    require_once("connection.php");
```

Comment:
What is our requirement for this course? is `Advance PHP` included?

Reply:
Yes, PHP Advance is included.

![DBinPHPCourseIntroduction](https://user-images.githubusercontent.com/38550259/64307251-1426b280-cfc8-11e9-9560-c033c06b4ae0.jpg)